client/blocks/upsell-nudge/index.jsx

```diff
@@ -56,11 +56,9 @@ export const UpsellNudge = ( {
 		return null;
 	}
 
-	if ( ! href && site ) {
-		href = addQueryArgs( { feature, plan }, `/plans/${ site.slug }` );
-	}
+	const link = href && site ? href : addQueryArgs( { feature, plan }, `/plans/${ site.slug }` );
```

Comment:
Not sure if these changes are intended, but previously we were setting a link to the plans page only when a site was defined. Now we're overriding the given href if no site is selected (i.e. when visiting the [Reader](https://github.com/Automattic/wp-calypso/blob/e7f164a8ef1fa81c0f15a626eb410781e051408b/client/reader/sidebar/reader-sidebar-nudges/index.jsx#L27-L38)), probably causing a broken link to `/plans/undefined` or similar.

Reply:
Whoops, that's unintentional. Fixed :)